script/dryrun

```diff
@@ -54,26 +54,32 @@ SKIP = {
     "sql/telemetry_derived/addons_v3/query.sql",
     "sql/telemetry_derived/addons_aggregates_v3/query.sql",
     # HTTP Error 408: Request Time-out
-    "sql/telemetry/clients_histogram_aggregates_v1/query.sql",
-    "sql/telemetry/clients_scalar_aggregates_v1/query.sql",
+    "sql/telemetry_derived/clients_histogram_aggregates_v1/query.sql",
+    "sql/telemetry_derived/clients_scalar_aggregates_v1/query.sql",
 }
 
 USE_DERIVED_DATASETS = {
     "sql/telemetry_derived/ssl_ratios_v1/query.sql",
     "sql/telemetry_derived/clients_last_seen_v1/query.sql",
     "sql/telemetry_derived/clients_daily_v7/query.sql",
     "sql/telemetry/fxa_users_last_seen_raw_v1/query.sql",
-    "sql/telemetry/clients_daily_keyed_boolean_aggregates_v1/query.sql",
-    "sql/telemetry/clients_daily_scalar_aggregates_v1/query.sql",
-    "sql/telemetry/client_histogram_probe_counts_v1/query.sql",
-    "sql/telemetry/clients_scalar_aggregates_v1/query.sql",
-    "sql/telemetry/clients_histogram_aggregates_v1/query.sql",
-    "sql/telemetry/clients_daily_keyed_scalar_aggregates_v1/query.sql",
+    "sql/telemetry_derived/clients_daily_keyed_boolean_aggregates_v1/query.sql",
+    "sql/telemetry_derived/clients_daily_scalar_aggregates_v1/query.sql",
+    "sql/telemetry_derived/clients_daily_histogram_aggregates_v1/query.sql"
+    "sql/telemetry_derived/clients_histogram_aggregates_v1/query.sql",
+    "sql/telemetry_derived/clients_scalar_aggregates_v1/query.sql",
+    "sql/telemetry_derived/clients_daily_keyed_scalar_aggregates_v1/query.sql",
     "sql/telemetry/firefox_accounts_exact_mau28_raw_v1/query.sql",
     "sql/telemetry/core_clients_daily_v1/query.sql",
-    "sql/telemetry/client_scalar_probe_counts_v1/query.sql",
     "sql/telemetry/firefox_kpi_dashboard_v1/query.sql",
     "sql/telemetry/core_clients_last_seen_raw_v1/query.sql",
+    "sql/telemetry/clients_daily_scalar_aggregates_v1/view.sql",
+    "sql/telemetry/clients_daily_histogram_aggregates_v1/view.sql",
+    "sql/telemetry/clients_scalar_aggregates_v1/view.sql",
+    "sql/telemetry/clients_histogram_aggregates_v1/view.sql",
+    "sql/telemetry/client_probe_counts_v1/view.sql",
+    "sql/telemetry_derived/clients_scalar_probe_counts_v1/query.sql",
+    "sql/telemetry_derived/clients_histogram_probe_counts_v1/query.sql",
```

Comment:
These can all be removed from this USE_DERIVED_DATASETS dict. We should let them hit the default shared-prod endpoint. I just checked and it looks like they all dry run successfully there.